metricbeat/_meta/beat.full.yml

```diff
@@ -7,6 +7,21 @@
 # You can find the full configuration reference here:
 # https://www.elastic.co/guide/en/beats/metricbeat/index.html
 
+#============================  Config Reloading ===============================
+
+# Config reloading allows to dynamically load modules. Each file which is
+# monitored must contain one or multiple modules as a list.
+metricbeat.reload:
+
+  # Glob pattern for configuration reloading
+  path: config/*.yml
```

Comment:
Changed to `conf.d`. I like the idea of creating this automatically. Reminds me to check to what the path is relative. Should the default be `${path.conf}/conf.d/*.yml`?